pydis_site/apps/api/viewsets/bot/nomination.py

```diff
@@ -282,12 +287,15 @@ def partial_update(self, request: HttpRequest, *args, **kwargs) -> Response:
             )
 
         # This is actually covered, but for some reason coverage don't think so.
```

Comment:
I think this comment can be removed, as coverage seems to detect the code fine now.

Reply:
That's true, I thought I removed it.
Yoinked !